draft-ietf-tls-tls13.md

```diff
@@ -4285,6 +4285,9 @@ client agrees to use this version, the negotiation will proceed as appropriate
 for the negotiated protocol. A client resuming a session SHOULD initiate the
 connection using the version that was previously negotiated.
 
+Note that a 0-RTT TLS 1.3 ClientHello is not compatible with older servers.
+See {{zero-rtt-backwards-compatibility}}.
```

Comment:
I think this is excessive redundancy. (see comment below on sticking the new text under this section as a subsection)